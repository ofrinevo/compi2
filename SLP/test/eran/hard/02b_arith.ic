class IFS { 

	static void main (string[] args) { 
	    int a = 5;
		int b = 10;
		int c = 20;

		Library.printi(a+b+c);  // should print 35
		Library.println("");

		Library.printi(a * b + c); // should print 70
		Library.println("");

		Library.printi((b / a) * c + c * c); // should print 440
		Library.println("");

		Library.printi(c - b - a - 5); // should print 0
		Library.println("");

	}
}

