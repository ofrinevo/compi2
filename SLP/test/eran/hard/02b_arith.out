35
70
440
0
